### PR TITLE
chore(phrases): translate phrases

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

- change GPT model to `gpt-3.5-turbo-1106` as it performs way better
- translate all untranslated phrases

## Testing

CI

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

**Comment by joshuahigginson1** [comment]
Saw this PR before I could comment on the issue ~ A brill translation service for loads of different languages: https://huggingface.co/spaces/facebook/seamless_m4t

**Review by simeng-li — APPROVED** [review]

File: packages/phrases/src/locales/de/errors/oidc.ts [hunk 1]
```diff
@@ -1,29 +1,24 @@
...
-  insufficient_scope: 'Token missing scope `{{scope}}`.',
+  insufficient_scope: 'Token fehlt den Anwendungsbereich `{{scope}}`.',
   invalid_request: 'Anfrage ist ungültig.',
-  invalid_grant: 'Grant request ist ungültig.',
+  invalid_grant: 'Grant Anfrage ist ungültig.',
   invalid_redirect_uri:
     '`redirect_uri` stimmt nicht mit den registrierten `redirect_uris` des Clients überein.',
   access_denied: 'Zugang verweigert.',
-  invalid_target: 'Ungültiger resource indicator.',
+  invalid_target: 'Ungültiger Ressourcenindikator.',
```
Comment (simeng-li) [inline packages/phrases/src/locales/de/errors/oidc.ts:12]: Should we worry about these capitalization issues?

File: packages/phrases/src/locales/de/translation/admin-console/tenants.ts [hunk 2]
```diff
@@ -13,26 +13,22 @@ const tenants = {
       'Geben Sie den Mandanten-Namen ein und sehen Sie Ihre Daten in der gehosteten Region und mit dem Umgebungstag an.',
     tenant_id: 'Mieter-ID',
     tenant_name: 'Mietername',
-    /** UNTRANSLATED */
-    tenant_region: 'Data hosted region',
-    /** UNTRANSLATED */
-    tenant_region_tip: 'Your tenant resources are hosted in {{region}}. <a>Learn more</a>',
+    tenant_region: 'Gehostete Region der Daten',
+    tenant_region_tip:
+      'Ihre Mandantenressourcen werden in {{region}} gehostet. <a> Mehr erfahren </a>',
```
Comment (darcyYe) [inline packages/phrases/src/locales/de/translation/admin-console/tenants.ts:18]: leading and trailing spaces in link?

File: packages/phrases/src/locales/de/translation/admin-console/tenants.ts [hunk 3]
```diff
@@ -13,26 +13,22 @@ const tenants = {
...
     environment_tag: 'Umgebungsmarke',
     environment_tag_description:
       'Tags verändern den Service nicht. Sie dienen lediglich zur Unterscheidung verschiedener Umgebungen.',
     environment_tag_development: 'Entw',
     environment_tag_staging: 'Staging',
     environment_tag_production: 'Prod',
-    /** UNTRANSLATED */
     development_description:
-      'Development environment is mainly used for testing and include all pro features but have watermarks in the sign in experience. <a>Learn more</a>',
+      'Die Entwicklungsumgebung wird hauptsächlich zum Testen verwendet und enthält alle Pro-Funktionen, weist jedoch Wasserzeichen im Anmeldeerlebnis auf. <a> Mehr erfahren </a>',
```
Comment (darcyYe) [inline packages/phrases/src/locales/de/translation/admin-console/tenants.ts:26]: ditto

File: packages/phrases/src/locales/de/translation/admin-console/tenants.ts [hunk 4]
```diff
@@ -44,57 +40,42 @@ const tenants = {
...
+      'Zögern Sie nicht, uns zu kontaktieren, wenn Sie Hilfe benötigen! Vielen Dank, dass Sie Logto gewählt haben!',
   },
   delete_modal: {
     title: 'Mieter löschen',
     description_line1:
-      'Möchten Sie Ihren Mieter "<span>{{name}}</span>" mit Umgebungssuffix-Tag "<span>{{tag}}</span>" wirklich löschen? Der Vorgang kann nicht rückgängig gemacht werden und führt zur dauerhaften Löschung aller Ihrer Daten und Kontoinformationen.',
+      'Möchten Sie wirklich Ihren Mandanten "<span>{{name}}</span>" mit Umgebungssuffix-Tag "<span>{{tag}}</span>" löschen? Dieser Vorgang kann nicht rückgängig gemacht werden und führt zur dauerhaften Löschung aller Ihrer Daten und Kontoinformationen.',
     description_line2:
-      'Bevor Sie Ihr Konto löschen, können wir Ihnen vielleicht helfen. <span><a>Kontaktieren Sie uns per E-Mail</a></span>',
+      'Bevor Sie Ihren Mieter löschen, können wir Ihnen vielleicht helfen. <span><a> Kontaktieren Sie uns per E-Mail </a></span>',
```
Comment (darcyYe) [inline packages/phrases/src/locales/de/translation/admin-console/tenants.ts:78]: leading and trailing spaces

File: packages/phrases/src/locales/de/translation/admin-console/upsell/paywall.ts [hunk 5]
```diff
@@ -3,7 +3,6 @@ const paywall = {
     '{{count, number}} Anwendung von <planName/> erreicht. Plan upgraden, um den Bedürfnissen Ihres Teams gerecht zu werden. Für Unterstützung können Sie uns gerne <a>kontaktieren</a>.',
   applications_other:
     '{{count, number}} Anwendungen von <planName/> erreicht. Plan upgraden, um den Bedürfnissen Ihres Teams gerecht zu werden. Für Unterstützung können Sie uns gerne <a>kontaktieren</a>.',
-  /** UNTRANSLATED */
```
Comment (darcyYe) [inline packages/phrases/src/locales/de/translation/admin-console/upsell/paywall.ts:6]: tag removed but not translated in this file

**Review by darcyYe — APPROVED** [review]
Approve to unblock

File: packages/phrases/src/locales/de/errors/oidc.ts (hunk 1 above)
Comment (gao-sun) [reply packages/phrases/src/locales/de/errors/oidc.ts:12]: actually i have no idea about if it's a language rule from Germany or we can use lowercase. i prefer to keep as it is

File: packages/phrases/src/locales/de/translation/admin-console/upsell/paywall.ts (hunk 5 above)
Comment (gao-sun) [reply packages/phrases/src/locales/de/translation/admin-console/upsell/paywall.ts:6]: you are the man

**Pull request merged**